sklearn/model_selection/_validation.py

```diff
@@ -577,14 +578,48 @@ def _score(estimator, X_test, y_test, scorer, is_multimetric=False):
 
 def _multimetric_score(estimator, X_test, y_test, scorers):
     """Return a dict of score for multimetric scoring"""
-    scores = {}
-
-    for name, scorer in scorers.items():
+    def _is_proba(x):
+        return isinstance(x, _ProbaScorer)
+
+    def _is_predict(x):
+        return isinstance(x, _PredictScorer)
+
+    # We want to keep the memmap and score types in a single
```

Comment:
I don't know what you mean by the memmap here.

Reply:
In the loop at https://github.com/scikit-learn/scikit-learn/pull/10979/files#diff-60033c11a662f460e1567effd5faa6f0R625

The scores are checked for being scalars and are unwrapped if they are memmaped.

`tmp_scores` contains the scores before being processed this way (it avoids doing the checks for each type of scorers)

Reply:
Ah, I now see what you mean. I think this makes sense to you as the one changing the code, but it would be hard for the reader to follow what you're referring to in this comment. It's better without the comment.